shared/css/course-blocks.scss

```diff
@@ -0,0 +1,187 @@
+@import 'color';
+@import 'font';
+
+/* HEADERS */
+
+.twolineheader {
+    h2 {
+      font-family: $gotham-regular;
+      font-weight: normal; 
+      color: $purple; 
+      font-size: 24px; 
+      margin-bottom: 0px; 
+      line-height: 30px;
+    }
+    h1 {
+      margin-top: 0px;
+      line-height: 30px; 
+      font-size: 30px;   
+      font-family: $gotham-bold; 
+      font-weight: normal;
+    }
+} 
+
+.coursesheader {
+  h1 + h2 {
+    margin-top: 10px;
+  }
+  h2 {
+    margin-bottom: 5px;
+  }
+}
+
+#landingpage h3 img.logo,
+.course_progress h3 img.logo {
+  height: 40px;
+}
+
+.betatext {
+  color: $realyellow;
+  background-color: $charcoal;
+  padding: 0 5px;
+}
+
+/* MINIBLOCKS */
+
+.span1 {  width: 60px;  }
```

Comment:
oh, doesn't this possibly compete with bootstrap's own definitions of these spans?  can we namespace them so they don't compete?